Extractor/FailureBuffer.cs

```diff
@@ -313,7 +313,7 @@ private async Task WriteEventsInflux(IEnumerable<UAEvent> events, CancellationTo
                     if (emitterState == null) continue;
                     eventBufferStates[group.Id] = bufferState = new InfluxBufferState(emitterState);
                 }
-                if (extractor.AllowUpdateState) bufferState.UpdateDestinationRange(group.Range.Min, group.Range.Max);
+                bufferState.UpdateDestinationRange(group.Range.Min, group.Range.Max);
```

Comment:
Turns out I was a bit overeager to use the `AllowUpdateState` field. This particular state should be updated no matter what...

I really want to nuke this feature too.

Reply:
maybe add a small note about why we're not checking for it here, so it doesn't look like an omission for someone reading it later?